app/javascript/react/components/ParkShowContainer.js

```diff
@@ -36,12 +98,69 @@ const ParksShowContainer = props => {
         rating={parkInfo.rating}
         description={parkInfo.description}
         photo={parkInfo.photo}
+        addNewReview={addNewReview}
       />
     }
 
```

Comment:
in what circumstance would `parkInfo` not be truthy? lets just move this into the return